libs/wire-api-federation/src/Wire/API/Federation/API/Galley.hs

```diff
@@ -250,18 +251,31 @@ data RemoteMLSMessageResponse
   deriving stock (Eq, Show, Generic)
   deriving (ToJSON, FromJSON) via (CustomEncoded RemoteMLSMessageResponse)
 
-data MessageSendRequest = MessageSendRequest
+data ProteusMessageSendRequest = ProteusMessageSendRequest
```

Comment:
For next time: maybe worth doing this as laying ground for the change, on a separate PR, to simplify review.

Reply:
That's a good point! Thanks! I'll make this into a separate PR to `develop` and then merge that back into the MLS branch; otherwise we'll be facing conflicts with this in every MLS PR.

Reply:
This has been done in PR https://github.com/wireapp/wire-server/pull/2925.